### PR TITLE
Clean up a bunch of the redundant code

### PR DESCRIPTION
**Pull request by Tastyfish**
Cleans up loads of redundant code in the `setCam` and `spawnPlayer` event handlers, so it's more coherent and more clear what's different between the spawn types.

### CODE REVIEW

File: client.lua [hunk 1]
```diff
@@ -150,7 +150,7 @@ RegisterNUICallback('spawnplayer', function(data)
         PostSpawnPlayer()
     elseif type == "normal" then
         local pos = QB.Spawns[location].coords
-        preSpawnPlayer()
+        PreSpawnPlayer()
```
Comment (FreakOutModz) [inline client.lua:153]: wrong, text always is `preSpawnPlayer`. Never start with a capital letter.

File: client.lua [hunk 2]
```diff
@@ -132,6 +96,24 @@ RegisterNUICallback('chooseAppa', function(data)
     SetEntityVisible(PlayerPedId(), true)
 end)
 
+local function PreSpawnPlayer()
+    SetDisplay(false)
+    DoScreenFadeOut(500)
+    Citizen.Wait(2000)
+end
+
+local function PostSpawnPlayer(ped)
```
Comment (FreakOutModz) [inline client.lua:105]: Please change typo to `postSpawnPlayer`

File: client.lua [hunk 3]
```diff
@@ -132,6 +96,24 @@ RegisterNUICallback('chooseAppa', function(data)
     SetEntityVisible(PlayerPedId(), true)
 end)
 
+local function PreSpawnPlayer()
```
Comment (FreakOutModz) [inline client.lua:99]: Please change typo as mentionned `preSpawnPlayer`

File: client.lua [hunk 4]
```diff
@@ -199,15 +159,7 @@ RegisterNUICallback('spawnplayer', function(data)
...
-        FreezeEntityPosition(ped, false)
-        RenderScriptCams(false, true, 500, true, true)
-        SetCamActive(cam, false)
-        DestroyCam(cam, true)
-        SetCamActive(cam2, false)
-        DestroyCam(cam2, true)
-        SetEntityVisible(PlayerPedId(), true)
-        Citizen.Wait(500)
-        DoScreenFadeIn(250)
+        PostSpawnPlayer()
```
Comment (FreakOutModz) [inline client.lua:162]: Change typo here

**Review by FreakOutModz — CHANGES_REQUESTED** [review]

**Comment by Tastyfish** [comment]
@FreakOutModz Sorry, I never got notified of the comments in this PR. As for your comments, is this a convention rule for local functions? If so, `qb-core` itself doesn't follow that convention.

**Comment by FreakOutModz** [comment]
Is not a real "rule", but is how should code look. `Qb-core` didn't follow this way but we can change everything easily!

**Comment by Tastyfish** [comment]
Ah, fair enough. I'm fine with any coding guideline rules, I was just legitimately confused.

**Comment by zfbx** [comment]
@Tastyfish I know this conversation is long over I know but to clarify it's a best practice to give local variables a lowercase first character and use an uppercase letter for global variables to easily be able to distinguish between them

**Pull request merged**